koku/masu/openapi.json

```diff
@@ -414,7 +414,7 @@
         "UpdateChargeGetResponse": {
```

Comment:
```suggestion
        "UpdateCostModelCostGetResponse": {
```

Reply:
If we update the above line - we also have to update line 335 in the openapi.json as well.